CHANGELOG.md

```diff
@@ -5,6 +5,185 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.0.0-beta.0
+
+<img width="100%" alt="MUI X v8 Beta is live" src="https://github.com/user-attachments/assets/61ec4dd8-c946-456b-8b45-d51de8772f5d">
+
+_Mar 18, 2025_
+
+We'd like to offer a big thanks to the 22 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Add Time Range Picker component
+- 🎁 Add support for `@mui/material` version 7 in all X packages
+- 🐞 Bugfixes
+- 🌍 Improve Chinese (zh-CN), (zh-HK), (zh-TW), Czech (cs-CZ), Korean (ko-KR) and Slovak (sk-Sk) locales on the Data Grid
+- 🌍 Improve Chinese (zh-CN), (zh-HK) and (zh-TW) locales on the Pickers
+
+## Breaking changes
+
+- ℹ️ The peer dependency on `@mui/material` has been updated to accept only v7.
+  This has been done to increase the adoption rate of ESM.
+  Since only v7 of `@mui/material` has proper ESM support, we decided to help with its adoption and fix numerous issues using X packages in environments where transpiling is not an option.
+
+Special thanks go out to the community members for their valuable contributions:
+@Blake-McCullough, @hlavacz, @k-rajat19, @layerok, @nusr, @owais635, @yelahj.
+Following are all team members who have contributed to this release:
+@alexfauquette, @arminmeh, @bernardobelchior, @cherniavskii, @DiegoAndai, @flaviendelangle, @Janpot, @JCQuintas, @KenanYusuf, @LukasTy, @MBilalShafi, @michelengelen, @noraleonte, @romgrk.
+
+<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->
+
+## Alpha release highlights
+
+Below are the highlights of the alpha releases leading up to this beta release:
+
+- 🚀📊 **New Pro Chart**: It is now possible to create Funnel charts—perfect for visualizing conversions, sales pipelines and more!
```

Comment:
Maybe we could group thos highlight by package and group / reword a bit.
For example the funnel and the radar wording are very different, they could probably be a single line.

Reply:
Not sure, since we have some things shared.

I was just proposing a very basic cleaning of the list, something like:


- React 19 support
- Highlight Data Grid 1
- Highlight Data Grid 2
- ...
- Highlight Data Grid N
- Highlight Chart 1 
- etc...

Maybe with the same icon every time per product line (📊 for the charts)

And merge lines that can be merged to have a more readable list:
For example:

- 📊 New charts: [Funnel](/) (pro plan only) and [Radar(/)

Or if we want to keep two lines;

- 📊 New pro chart: [Funnel](/)
- 📊 New community chart: [Radar](/)

Reply:
WDYT about the updates: [Refactor alpha highlights](https://github.com/mui/mui-x/pull/17020/commits/e8449e4e3fb793298bb3bf379deb75ae9baf8507)? 🤔

Reply:
I like this new strategy of using 📊 for all charts update :+1: